automate-gitops-pull-request.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# Please Use Google Shell Style: https://google.github.io/styleguide/shell.xml
+
+# ---- Start unofficial bash strict mode boilerplate
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -o errexit  # always exit on error
+set -o errtrace # trap errors in functions as well
+set -o pipefail # don't ignore exit codes when piping output
+set -o posix    # more strict failures in subshells
+# set -x          # enable debugging
+
+IFS=$'\n\t'
+# ---- End unofficial bash strict mode boilerplate
+
+error=""
+for VAR in SERVICE DOCKER_REPOSITORY GITHUB_TOKEN GH_USERNAME GH_EMAIL REACTION_GITOPS_REVIEWERS; do
+  if [[ -z "${!VAR}" ]]; then
+    error="${error}Missing required environment variable: ${VAR}\n"
+  fi
+done
+if [[ -n "${error}" ]]; then
+  echo -e "${error}" 1>&2
+  exit 1
+fi
+
+# Install Kustomize
+./install-kustomize.sh
+
+# Install Hub
+./install-hub.sh
+
+# Clone reaction-gitops repository and configure username and email for signing off commits
+hub clone https://"${GITHUB_TOKEN}"@github.com/reactioncommerce/reaction-gitops.git
```

Comment:
Minor nit but I think quotes in the middle of an argument, while fine and correct, are hard to read, so I always just put them an the start/end like `"https://${SOME_VAR}/some/junk/here?foo${ANOTHER_VAR}"`. There's a bunch of additional instances of this in the code below too.

Reply:
Updated